pbpy/pbunreal.py

```diff
@@ -585,7 +585,7 @@ def get_bundle_verification_file(bundle_name):
 
 @lru_cache()
 def get_engine_base_path():
-    root = get_engine_install_root()
+    root = get_engine_install_root(False)

```

Comment:
This breaks a lot of user flows. Your CI machine should define CI=1, and that _should_ make the `get_engine_install_root` to not prompt.

Reply:
Ahh, that makes sense. I'll revert this change!

Reply:
I've dropped the commit. Though I did run into an issue where, once CI=1 is set, it would *only* look for UE4 inside `C:\ue4`.

Reply:
Refer to .ue4v-user-ci to see how environment variables can be set for CI for configuration

Reply:
Okay I've sort of gotten it working. It's failing in `get_engine_base_path` as it's an EGS install. I've added a function that checks for `.egstore` similar to `is_source_install`. Do you mind if this is added? On the build machines I have, this works fine as we don't have any specific requirements that would require us to build the engine ourselves, plus PBSync works fine with marketplace builds.

Reply:
But get_engine_base_path already has handling for installed builds, are you saying this behavior no longer works?

Reply:
It doesn't work, as that code is run when if root is none, which in my case it isn't. As it doesn't fulfil the other checks below, source install, longtail, get engine version, `get_engine_base_path` returns none.

Reply:
Why isn't it none if you aren't using a source install?

Reply:
`get_engine_install_root` checks for the dirs in the config, it gets them fine and returns that path, so `root` is set correctly to the path I supplied to it